modules/API/API.js

```diff
@@ -107,8 +108,24 @@ function initCommands() {
 
         return false;
     });
-    transport.on('request', ({ name }, callback) => {
+    transport.on('request', (request, callback) => {
+        const { name } = request;
+
         switch (name) {
+        case 'invite':
+            APP.store.dispatch(
+                sendInvitesForItems(request.invitees))
+                .then(failedInvites => {
+                    const failed = failedInvites.length === 0;
```

Comment:
`const failed = failedInvites.length === 0;` doesn't look right. I think it should be `const failed = failedInvites.length;`